README.md

```diff
@@ -19,14 +17,28 @@ Dependencies:
 * Virtualbox
 * virtualization enabled in bios
 
-On Linux systems:
+Procedure:
+
+1. Clone the pelux-manifests git repository.
+2. Start vagrant
 ```bash
-MANIFEST=<manifest> BRANCH=<branch> vagrant up
+MANIFEST=<manifest> vagrant up
 ```
 
+Note:
+The virtual machine started via vagrant will sync the cloned git repository and use the manifests contained in it
```

Comment:
Sorry if I've got this wrong but shouldn't the manifest here be in singular form? Or did I misunderstand something.